src/pages/library/DataBrowser.js

```diff
@@ -55,12 +57,12 @@ const extractCatalogFilters = dataCatalog => {
     labels: getUnique(dataset => getConsortiumsFromDataset(dataset), dataCatalog)
   }, {
     name: 'Data use policy',
-    labels: getUnique('dataReleasePolicy.policy', dataCatalog),
+    labels: getUnique(dataset => getDataReleasePolicyFromDataset(dataset).policy, dataCatalog),
     labelRenderer: rawPolicy => {
-      const { label, desc } = datasetReleasePolicies[rawPolicy] || datasetReleasePolicies.releasepolicy_other
+      const { label, desc } = datasetReleasePolicies[rawPolicy] || datasetReleasePolicies.unknownReleasePolicy
       return [div({ key: rawPolicy, style: { display: 'flex', flexDirection: 'column' } }, [
-        label ? label : rawPolicy,
-        desc && div({ style: { fontSize: '0.625rem', lineHeight: '0.625rem' } }, [desc])
+        label,
+        rawPolicy !== datasetReleasePolicies.unknownReleasePolicy.policy && div({ style: { fontSize: '0.625rem', lineHeight: '0.625rem' } }, [desc])
```

Comment:
It seems to me that `unknownReleasePolicy` isn't buying us much here, and makes the code more awkward than if we used `undefined` directly.

Reply:
Ah! I just had an idea for a different approach to this that handles that better. Going to try to do that as a bigger change than just right here. (This approach was mainly to handle the case where we add something to the API that we haven't made the corresponding UI change to yet)